tests/validation.rs

```diff
@@ -12,14 +12,15 @@ use common::config::CONFIG;
 use models::{error_message::ErrorMessage, key, status::Status};
 use pretty_assertions::assert_eq;
 use reqwest::{blocking::Response, Method, StatusCode};
+use rstest::rstest;
 use serde_json::json;
-use test_case::test_case;
 use uuid::Uuid;
 
-#[test_case("0"; "Zero key size")]
-#[test_case("-8"; "Negative key size")]
-#[test_case("abc01"; "Alphanumeric key size")]
-fn key_size(key_size: &str) {
+#[rstest]
+#[case("0")] // "Zero key size")
```

Comment:
Remove the double quotes and replace the brace at the end of the line with a `.`.